pyccel/codegen/printing/fcode.py

```diff
@@ -1740,11 +1740,11 @@ def _print_AugAssign(self, expr):
         like   = expr.like
 
         if isinstance(op, AddOp):
-            rhs = PyccelAdd(lhs, rhs)
+            rhs = PyccelAdd(lhs, rhs, simplify = True)
         elif isinstance(op, MulOp):
             rhs = PyccelMul(lhs, rhs)
         elif isinstance(op, SubOp):
-            rhs = PyccelMinus(lhs, rhs)
+            rhs = PyccelMinus(lhs, rhs, simplify = True)
```

Comment:
These functions handle the `AugAssign` operator. The `AugAssign` operator handles the operators `+=`, `-=`, `*=`, etc.
As this is an assignment the `lhs` cannot be a `Literal`, so I don't think it is worth simplifying these expressions